### PR TITLE
Feature request: multiple URL support for getInfo()

### PR DESCRIPTION
**Pull request by tifroz**
...such as:

```javascript
var youtubedl = require('youtube-dl');
var url1 = 'http://www.youtube.com/watch?v=WKsjaOqDXgg';
var url2 = 'https://vimeo.com/6586873';
youtubedl.getInfo([url1, url2], function(err, info) {
  if (err) throw err;

  console.log('title for the url1:', info[0].title);
  console.log('title for the url2:', info[1].title);
});
```

### CODE REVIEW

**Comment by fent** [comment]
Is this different than having the user handle it because it only calls the process once?

**Comment by tifroz** [comment]
Yes, that is the only difference.

If I understand correctly how node's execFile works, this will result in
significant resource savings when calling getInfo on many urls at once
(spawning the process just once instead of 10 or 100 times)

File: README.md [hunk 1]
```diff
@@ -66,6 +66,19 @@ Running that will produce something like
     filename: Ace Rimmer to the Rescue-WKsjaOqDXgg.flv
     format id: 34
 
+You can use an array of urls to produce an array of response objects with matching array index (e.g. the 1st response object will match the first url etc...)
+```javascript
```
Comment (fent) [inline README.md]: new line above code block, also use `js` instead of `javascript` for consistency.

File: lib/youtube-dl.js [hunk 2]
```diff
@@ -85,36 +85,40 @@ ytdl.exec = function (url, args, options, callback) {
  * Calls youtube-dl with some arguments and the `callback`
  * gets called with the output.
  *
- * @param {String} url
+ * @param {String} urls (can be substituted for an {Array.<String>} when called from getInfo())
```
Comment (fent) [inline lib/youtube-dl.js]: use a `|` to specify multi types

    @param {String|Array.<String>}

File: lib/youtube-dl.js [hunk 3]
```diff
@@ -85,36 +85,40 @@ ytdl.exec = function (url, args, options, callback) {
...
  * @param {Array.<String>} args
  * @param {Array.<String>} args2
  * @param {Object} options
  * @param {Function(!Error, String)} callback
  */
-function call(video, args1, args2, options, callback) {
+function call(urls, args1, args2, options, callback) {
+  if (typeof(urls) === "string") {
```
Comment (fent) [inline lib/youtube-dl.js]: no parentheses around `typeof` for consistency

File: test/getFormat.js [hunk 4]
```diff
@@ -3,6 +3,7 @@ var ytdl   = require('..');
 var assert = require('assert');
 
 var youtubevideo = 'http://www.youtube.com/watch?v=0k2Zzkw_-0I';
+var vimeovideo = 'https://vimeo.com/6586873';
```
Comment (fent) [inline test/getFormat.js]: ?

**Comment by fent** [comment]
Ok looks mostly good, most of my comments are about styling format. It's good to keep everything using the same style so that the overall project code looks consistent.

And the tests fail because `ytdl.getExtractors()` calls `call()` with `null` for the url(s).

File: README.md (hunk 1 above)
Comment (tifroz) [inline README.md]: Not sure what the standard is? The README's standard is to use `javascript` up until this point, but is uses `js` after that

File: lib/youtube-dl.js (hunk 2 above)
Comment (tifroz) [inline lib/youtube-dl.js]: ah...will do, this is great

File: test/getFormat.js (hunk 4 above)
Comment (tifroz) [inline test/getFormat.js]: removed

File: README.md (hunk 1 above)
Comment (fent) [inline README.md]: good point.

Comment (tifroz) [inline README.md]: I changed all the `js` into `javascript` in the last version of the PR

**Comment by fent** [comment]
Thank you!

**Pull request merged**